language-support/java/codegen/src/main/scala/com/digitalasset/daml/lf/codegen/backend/java/inner/VariantRecordClass.scala

```diff
@@ -13,21 +13,22 @@ import scala.jdk.CollectionConverters._
 private[inner] object VariantRecordClass extends StrictLogging {
 
   def generate(
+      packageId: PackageId,
       typeParameters: IndexedSeq[String],
       fields: Fields,
       name: String,
       superclass: TypeName,
       packagePrefixes: Map[PackageId, String],
-  ): TypeSpec.Builder =
+  ): TypeSpec =
```

Comment:
All other methods returned a `TypeSpec` directly and there didn't seem to be there a good reason (anymore?) to return a `Builder` here specifically.